src/engine/renderer/tr_image.cpp

```diff
@@ -2038,122 +2099,109 @@ image_t        *R_FindCubeImage( const char *imageName, int bits, filterType_t f
 		pic[ i ] = nullptr;
 	}
 
-	for ( i = 0; i < 6; i++ )
+	for ( f = 0; f < numMultifileCubeMapFormats; f++ )
```

Comment:
Optional nit: Instead of having an index it's nicer to use a for-each loop like `for (const multifileCubeMapFormat_t format : multifileCubeMapFormats)`

Reply:
oh, interesting!